### PR TITLE
Example security: nginx.conf

### PR DESCRIPTION
**Pull request by emtudo**

### CODE REVIEW

**Comment by vinicius73** [comment]
+1

File: nginx.conf.example [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  root          /var/www/october;
+  index         index.php;
+
+  # Deny access to dotfiles
+  location ~ /\. {
+    deny  all;
+  }
+
+  # Deny access to framework directories
+  location ~ /(bootstrap/|config/|modules|plugins/|storeage/|themes/|vendor/) {
```
Comment (jwilson8767) [inline nginx.conf.example]: "storage" not storeage

**Comment by emtudo** [comment]
fixed!

File: nginx.conf.example (hunk 1 above)
Comment (emtudo) [inline nginx.conf.example]: fixed! thanks!

**Comment by daftspunk** [comment]
These improvements should be made to [the documentation](https://github.com/octobercms/docs/blob/master/setup-configuration.md#nginx-configuration).

This is not something we really want to include in the core files.

**Pull request closed without merging**